Moderation-Policy.md

```diff
@@ -195,8 +195,8 @@ Scenario 1:
 
 Scenario 2:
  * A non-Collaborator posts a comment that is against the Code of Conduct.
- * A Collaborator sees the comment and asks the author to self-moderate.
- * The author refuses to self-moderate.
+ * A Collaborator sees the comment and asks the author to edit it.
```

Comment:
```suggestion
 * A Collaborator sees the comment and asks the author to edit it, so that it aligns with the Code of Conduct.
```

Reply:
I'd prefer this kind of thing to be a separate PR. There's a bigger discussion to be had here. (For example: Does the comment for which editing is being requested need to be a clear violation of CoC or a grey area or something that just doesn't seem to align with the spirit but technically complies or what? I'm opposed to anything that encourages hair-splitting and rule-lawyering on this stuff, so I'm actually against mentioning the Code of Conduct here. But I might be in the minority on that for sure.)

Reply:
Not directly related to this suggestion, but adjacent to it: I would encourage my fellow moderators to err on the side of a less formal style when dealing with other collaborators. If someone leaves a cranky comment, I'd much rather have a moderator say "Hey, can you edit the stuff in that comment that might be interpreted as insulting?" rather than "Please edit that comment to align with the Code of Conduct." Bringing the Code of Conduct into things escalates them. That's fine if your goal is to lay down the law for someone. But in most cases, we're just trying to encourage people to be more friendly and not let their irritation at something feed a vicious cycle.

The exception, of course, is if something is a blatant CoC violation, but honestly, in that case, I'm not sure that asking people to edit their own comments is the right process anyway. But again, this is a much bigger discussion than simply removing the term "self-moderate".